crates/cubecl-linalg/src/convolution/loader/im2col_tma.rs

```diff
@@ -116,3 +165,27 @@ impl<MP: MatmulPrecision, G: ConvGemmConfig> TmaIm2colLoader<MP, G> {
         TmaIm2colReader::<MP>::new(*this.stages.index(stage_idx), InputIdent::Lhs)
     }
 }
+
+#[cube]
+pub(crate) fn div_mod_seq(pos: u32, shape: &Sequence<FastDivmod>) -> (u32, Sequence<u32>) {
```

Comment:
What is the output sequence?

Reply:
Didn't think it would need a comment because it's for internal use in the convolution kernels only. It decomposes the linear `m` index into the spatial (depth, height, width) indices and a remainder (the batch). So the sequence is the coordinate along each of the dimensions in the `Sequence`. It's just a generic version of
```rust
let (rem, out_x) = (pos / width, pos % width);
let (rem, out_y) = (rem / height, rem % height);
(rem, [out_y, out_x])
```
In that example, the sequence would contain `[height, width]`.

Reply:
I would add your comment in the code, just for clarity if we have to update/change the code later as context.